docs-issue-generation: added fix for abrupt script failure

mohini-crl opened the pull request:
Description - We had an issue raised by Rich  where doc issues for few PR wasn't created because of abrupt failure in the script due to not finding SHA or pr number in the JIRA issue.

Fix - I have added a condition where even if SHA or PR number isn't found for one issue , it should still continue the execution for other PRs so that doc issues for them is created


Epic:none
Part of https://cockroachlabs.atlassian.net/browse/DOC-10884

mohini-crl committed:
docs-issue-generation: added fix for abrupt script failure in case commit info in not found

This script used to break whenever there was doc issue found with no epic information and in turn it stopped further execution of doc issue creation for eligible PR. Have added continue in such cases so that execution for eligible PRs can continue

Epic:none

cockroach-teamcity commented:
[![CLA assistant check](https://cla.crdb.dev/pull/badge/not_signed)](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=129628) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=129628) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.crdb.dev/check/cockroachdb/cockroach?pullRequest=129628) it.</sub>

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/129628)
<!-- Reviewable:end -->

ebembi-crdb reviewed (approved):
Looks good to me!

mohini-crl committed:
docs-issue-generation: added fix for abrupt script failure in case commit info in not found

This script used to break whenever there was doc issue found with no epic information and in turn it stopped further execution of doc issue creation for eligible PR. Have added continue in such cases so that execution for eligible PRs can continue

Epic:none

mohini-crl commented:
bors r+
